docs/source/data/data_catalog_yaml_examples.md

```diff
@@ -365,7 +365,7 @@ airplanes:
 
 In this example, the default `csv` configuration is inserted into `airplanes` and then the `load_args` block is overridden. Normally, that would replace the whole dictionary. In order to extend `load_args`, the defaults for that block are then re-inserted.
 
-## Read the same file using two different datasets
+## Transcode dataset: Read the same file using two different datasets
```

Comment:
```suggestion
## Transcoding datasets: Read the same file using different datasets
```

Reply:
I left `Transcode dataset` as it was in the previous docs version, though I agree with the suggestion. Will update.